connect-widget/app/src/shared/localStorage/providers.ts

```diff
@@ -12,11 +12,14 @@ const providers: DefaultProvider[] = [
 const providersLocalStorageKey = "providers";
 
 
-export const getProviders = (searchProviders: string | null): DefaultProvider[] => {
+export const getProviders = (searchProviders: string | null, isDemo: boolean): DefaultProvider[] => {
   const validProviders = getValidProviders(searchProviders);
 
   if (validProviders) {
-    localStorage.setItem(providersLocalStorageKey, JSON.stringify(validProviders));
+    if (!isDemo) {
```

Comment:
should this be `if (!isDemo && validProviders)` ?

Reply:
Same issue as the colors. I could create a separate if check specifically for the localstorage

Reply:
ditto